plugins/splitwindow.c

```diff
@@ -318,10 +318,15 @@ static void split_view(gboolean horizontal)
 	gtk_container_add(GTK_CONTAINER(pane), notebook);
 	g_object_unref(notebook);
 
+	/* used just to get us to the same visual height level as the main editor window */
+	viewport = gtk_viewport_new(NULL, NULL);
```

Comment:
This is however the case where we need to show the shadow (but don't want the frame in addition to it) so we need something.

I have to check which how the _shadow_ (not frame) looks around GtkViewport and GtkScrolledWindow.